analytics/core_test.go

```diff
@@ -0,0 +1 @@
+package analytics
```

Comment:
Should delete this file unless you're planning on adding to it.

Reply:
tests coming up, once the code looks good enough